documentation/src/components/Constants.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
```

Comment:
hmm thanks for adding these. interesting pattern/different pattern than im used to. 

So we're already in a components folder, which to my understanding is the equivalent of constants. we may want to put each of these in their own separate file..and import them that way.

For example:

src/components/DesktopProviderSetup.js and it would only have the desktop provider component 

or 
src/components/ModelSelectionTip.js